src/o2tuner/run.py

```diff
@@ -60,20 +85,68 @@ def run_optimisation(cwd, config):
     return ret
 
 
-def run_stages(config, which_stages):  # noqa: C901
+def run_inspector(cwd, config, stages_optimisation):
+    """
+    Wrapper to run the optimisation.
+    """
+    func_name = config.get("entrypoint", None)
+    if not func_name or "file" not in config:
+        print("ERROR: Need path to python file and name of function entrypoint")
+        return False
+    func = import_function_from_file(config["file"], func_name)
+    if not func:
+        return False
+    # which optimisations to load
+    if "optimisations" not in config:
+        print("ERROR: Need key \"optimisations\" to know which optimisations to load")
+        return False
+
+    user_config = config.get("config", {})
+
+    inspectors = []
+
+    for optimisation in config["optimisations"]:
+        if optimisation not in stages_optimisation:
+            print(f"WARNING: Optimisation stage {optimisation} not defined, cannot construct inspector for that. Skip...")
+            continue
+        opt_config = stages_optimisation[optimisation]
+        opt_cwd, _ = get_stage_work_dir(optimisation, opt_config)
+        optuna_config = get_optuna_config(optimisation, opt_config)
+        insp = O2TunerInspector()
+        if not insp.load(optuna_config, opt_cwd, user_config):
+            continue
+        inspectors.append(insp)
+
+    if not inspectors:
+        print("WARNING: No O2TunerInspectors loaded, nothing to do")
+        return False
+
+    # cache current directory
+    this_dir = getcwd()
+    # change to this cwd and afterwards back
+    chdir(cwd)
+    ret = func(inspectors, user_config)
+    chdir(this_dir)
+    return ret
+
+
+def run_stages(config, which_stages):  # noqa: C901 pylint: disable=too-many-branches
```

Comment:
Just a comment: I see increasing number of disabled check for QA and pylint. I personally don't like to do this kind of patching issues, either one realises that they are unneeded/wrong/too strict and re-set the tests or it does not make any sense at all to have checks.

Reply:
It's true what you say and I agree.
My proposal:
Let's keep it as is for the moment. 
I think once you have your overall-entry-point development, I suggest we do the code sprint to put the code into a preliminary - say - `v0.1.0` version.
And there we discuss logging, `pylint`/`flake8` settings etc #22 
What do you think?